tests/test_wave_05.py

```diff
@@ -73,27 +73,42 @@ def test_create_goal(client):
 
 @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    pass
+    # pass
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goal/1", json={
+        "title": "Updated Goal Title",
+        "description": "Updated Test Description"
+    })
+    response_body = response.get_json()
 
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "task": {
+            "id": 1,
+            "title": "Update Goal Title",
+            "description": "Update Test Description",
+            "is_complete": False
+        }
+}
```

Comment:
The `put` request to update a goal return a dictionary of the `goal` updated. This assert should expect a similar dictionary rather than a `task`.